CondCore/ESSources/interface/DataProxy.h

```diff
@@ -23,28 +25,25 @@ namespace cond {
 }  // namespace cond
 
 template <class RecordT, class DataT, typename Initializer = cond::DefaultInitializer<DataT>>
-class DataProxy : public edm::eventsetup::DataProxyTemplate<RecordT, DataT> {
```

Comment:
Outside of CondDBESSource, the class DataProxyTemplate is only used in Core tests. Do we still need it? Possibly it is useful enough for the tests that we still need it to support the tests? Or do we envision needing it for some other purpose in the future?

Reply:
Given the complexity of the code, I wanted to keep it in FWCore so it is Core responsibility. To that end, I also made sure it had a test.